examples/Interactive Widgets/Exploring Graphs.ipynb

```diff
@@ -78,748 +78,11 @@
-       "3ryZm2++2aFwzc7OplevXnTs2JGEhASFq4iDFLAiV8Gtt97K7t27Wb9+PRERERQXF1dY1tvbmxkz\n",
-       "ZvDcc89dts9isdh1H9bRd1+//fZbAgICCA0NJSYmxq4l9ETkUgpYkaukZcuW7Nixg4yMDIKDgyud\n",
-       "f3j69OkkJyfz2WefXbavqvuwOTk5bN++nZCQELv6tWfPHnr37s1zzz3HrFmzNIGESDUpYEWuoiZN\n",
-       "mrBx40bc3d0ZOHBghbMyNW7cmFmzZjFnzpzL9lX1qs6qVasYNmwYnp6eVfZnw4YNDB8+nNjYWMLD\n",
-       "w+0+DxG5nAJW5Cpr2LAhq1evpkuXLvTq1Yvs7Oxyy02dOpUDBw6wd+/eS7ZXdonYGMPy5ct5+OGH\n",
-       "q+zHsmXLmDx5Mhs3biQwMNDxExGRSyhgRa4BLi4uvPbaa4wZM4aAgAC+/fbby8o0atSIqKgoZs+e\n",
-       "fcn2ygI2LS2Ns2fP0rt37wqPbYxh/vz5LFy4kI8++oh77rnnyk5GRAAFrMg1w2KxEBkZSVRUFL17\n",
-       "9+bjjz++rExYWBhZWVls27bt4rbK7sH+dqnXxaX8P/Xi4mKmTZvG2rVr2b17N+3bt6+ZkxERvQcr\n",
-       "ci3asGEDEyZM4O23377scu0777zD4sWL2bNnDxaLBWMMTZs2JTMzk2bNml0sV1BQQOvWrfn0009p\n",
-       "27btZcfIz89nzJgx5ObmkpiYSJMmTZx9WiL1ikawItegoUOHsm7duoshW1pISAh5eXls2LABqPhV\n",
-       "nXXr1tG5c+dyw/XUqVMMGDAAq9XKxo0bFa4iTqCAFblGde/eneTkZObMmcOiRYsuTkjh4uLC/Pnz\n",
-       "efbZZykpKQHg5ptvZs+ePaSnp5OTk4MxhtjY2HIfbsrKyqJHjx786U9/YuXKlbi7u9fqeYnUF7pE\n",
-       "LHKN++GHH7jvvvsYOHAgr7zyCi4uLhhj6NatGwMHDuSbTz9l4wcf0NTVFVvDhpwqKqK5lxc/njlD\n",
-       "ekYGN95448W2vvjiCwIDA5kxYwZPPPHEVTwrkeufAlakDjh9+jTDhg2jTZs2xMbG8vnnnxM8eDCW\n",
-       "nBxmGsNDwG8XeQ2QCrzq5sY2NzcemzGD56KjSUlJITg4mNdee43Ro0dfvZMRqScUsCJ1RH5+PqGh\n",
-       "oRw5coTDX3/NG3l5BAOVzbP0IzDSZqPRnXfy+aFDvPvuu/Tr16+WeixSvylgReqQ/fv38+d77mHd\n",
-       "+fNU/GbrpfKBXkDHkBBWvPuuE3snIqXpISeROmTWI4/wYplw/QWYCLQFPIEuwOZS+z2ALcDmdevK\n",
-       "ncBCRJxDAStSRxw6dIi0tDTKrolzHvADPgJ+Bl4ARgGHS5VpBjxcVMSSxYtrpa8iokvEInXGzEcf\n",
-       "xfWtt3ipqKjKsp2BucCIUtsyga5WK1nHj2Oz2ZzSRxH5fxrBitQRGxITGWNHuB4DvgM6ldneFujk\n",
-       "5kZqamrNd05ELqOAFakjTv78M75VlCkCxgLhwG3l7Pc1hpMnT9Z010SkHApYketECTAOaAT88yr3\n",
-       "RUQUsCJ1RnNPT36sYJ/hwpPEOcB7gGsF5X60WGjevLkzuiciZShgReqIoUFBrG7QoNx9fwG+AdYB\n",
-       "DSuonwl8ef48AQEBTumfiFxKAStSR0Q89hixrq4UlNl+GFgKfAa04sKUiU2Ad8qUe8vNjfFhYVit\n",
-       "Vud3VkT0mo5IXXJfQAAPpKYS4WC9U8AdHh58tH8/t99+uzO6JiJlaAQrUocsWrKEKJuNnQ7UyQdG\n",
-       "2GyMmzhR4SpSixSwInXInXfeybvr1vGg1coaLjzcVJkfgb5WK20CA3lZsziJ1CoFrEgd07dvX7ak\n",
-       "pDDXz4+7GjfmTeBsqf0G2AWMsVrp2KgR9z3+OCvWrMHFRX/uIrVJ92BF6ihjDDt27OB/Xn6ZTdu2\n",
-       "4dWgAQ1dXDhVVMQNzZszbeZMwiZMwMvL62p3VaReUsCKXAfy8vI4efIkhYWFeHl50bx5cyyWylaK\n",
-       "FRFnU8CKiIg4gW7KiIiIOIECVkRExAkUsCIiIk6ggBUREXECBayIiIgTKGBFREScQAErIiLiBApY\n",
-       "ERERJ1DAioiIOIECVkRExAkUsCIiIk6ggBUREXECBayIiIgTKGBFREScQAErIsRum1wAAACiSURB\n",
-       "VCLiBApYERERJ1DAioiIOIECVkRExAkUsCIiIk6ggBUREXECBayIiIgTKGBFREScQAErIiLiBApY\n",
-       "ERERJ1DAioiIOIECVkRExAkUsCIiIk6ggBUREXECBayIiIgTKGBFREScQAErIiLiBApYERERJ1DA\n",
-       "ioiIOIECVkRExAkUsCIiIk6ggBUREXECBayIiIgTKGBFREScQAErIiLiBP8HyD8bEel+oXYAAAAA\n",
-       "SUVORK5CYII=\n"
-      ],
-      "text/plain": [
-       "<matplotlib.figure.Figure at 0x1076c6e10>"
-      ]
-     },
-     "metadata": {},
-
```

Comment:
Do you remove output on purpose ? I though we would leave them for nbviewer ?

OK.